.github/dependabot.yml

```diff
@@ -48,3 +48,42 @@ updates:
     time: "03:00"
     timezone: Europe/Paris
   open-pull-requests-limit: 10
+- package-ecosystem: composer
+  directory: "/"
+  schedule:
+    interval: weekly
+    day: saturday
+    time: "03:00"
+    timezone: Europe/Paris
+  target-branch: stable25
```

Comment:
don't we also need to apply to stable24 like you did with npm?
In this PR you applied to:
|tool|branch|
|----|----|
|composer|stable25|
|npm|stbale25|
|npm|stable24|